save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityCommentTab.kt

```diff
@@ -48,21 +61,28 @@ val vulnerabilityCommentTab: FC<VulnerabilityCommentTabProps> = FC { props ->
         setComments(newComments)
     }
 
-    div {
-        className = ClassName("col-12 mt-4")
-        newCommentWindow {
-            addComment = { enrollCommentsRequest() }
-            currentUserInfo = user
-        }
-    }
-
-    comments.forEach { message ->
+    if (comments.size > DEFAULT_MAX_COMMENT_AMOUNT && !isAllCommentsShown) {
+        comments.take(2).display()
```

Comment:
could be extracted into variable

Reply:
you mean `comments.take(2)`? It could, but see no need in creating two variables: `comments.take(2)` and `comments.takeLast(2)`

Reply:
no, I mean just `2`

Reply:
Will replace with `DEFAULT_MAX_COMMENT_AMOUNT / 2` as it makes more sense